README.md

```diff
@@ -1,8 +1,8 @@
-<center><img src="https://redspread.com/images/logo.svg" alt="logo" width= "400"/>
+<p align="center"><img src="https://redspread.com/images/logo.svg" alt="logo" width="400" /></p>
```

Comment:
And while we're nitpicking, it's a "paragraph," not a "paraphrase": https://www.w3.org/TR/html-markup/p.html